sys/sysent.c

```diff
@@ -42,6 +43,7 @@ int sys_exit(thread_t *td, syscall_args_t *args) {
 
 /* clang-format hates long arrays. */
 sysent_t sysent[] = {
-  {sys_nosys}, {sys_exit},  {sys_open},  {sys_close}, {sys_read},  {sys_write},
-  {sys_lseek}, {sys_nosys}, {sys_nosys}, {sys_nosys}, {sys_fstat}, {sys_sbrk},
+  {sys_nosys}, {sys_exit},  {sys_open},  {sys_close}, {sys_read},
```

Comment:
Please use `[SYS_MMAP] = {sys_mmap}` extended C syntax to avoid problems with entry renumbering.